code/src/utest/plugin/lsttokens/choose/AbilityTokenTest.java

```diff
@@ -36,6 +36,9 @@
 import plugin.lsttokens.testsupport.CDOMTokenLoader;
 import plugin.qualifier.ability.PCToken;
 
+import org.junit.jupiter.api.Test;
+
+
```

Comment:
Extra newline

Reply:
I probably won't fix this until you specifically ask me too. Its hard to catch by hand auto-formatting is going to cause more unrelated changes.